xml/release-notes.xml

```diff
@@ -305,11 +305,37 @@
    Additionally, check <xref linkend="removed-deprecated"/>.
   </para>
 
-  <!-- <sect2 xml:id="sec.upgrade.152">
-   <title>Upgrading from &opensuseleap; 15.2</title>
+   
+  <sect2 xml:id="sec.upgrade.152">
+   <title>Seamless upgrade from &opensuseleap; 15.2</title>
+   <!-- bnc#1182629 -->
+   <para>
+   &opensuseleap; 15.3 is newly built on top of binary rpms from &sles;
+   This change was introduced as part of Closing The Leap Gap (CtLG) effort
+   to bring &opensuseleap; and &sles; closer together.
+
```

Comment:
Excess white space in `<para>` is removed so this is actually one big paragraph. You need to use `<para>` to make paragraphs.